optimum/bettertransformer/models/__init__.py

```diff
@@ -49,6 +50,7 @@ class BetterTransformerManager:
         "mbart": ("MBartEncoderLayer", MBartEncoderLayerBetterTransformer),
         "rembert": ("RemBertLayer", BertLayerBetterTransformer),
         "roberta": ("RobertaLayer", BertLayerBetterTransformer),
+        "roformer": ("RoFormerLayer", RoFormerLayerBetterTransformer),
```

Comment:
Great! Just something I noticed: I think the code for `BertLayerBetterTransformer` and the RoFormerLayerBetterTransformer you added is the same. Could you try here with this suggestion to see if it's fine? In which case the RoFormerLayerBetterTransformer is not needed.
```suggestion
        "roformer": ("RoFormerLayer", BertLayerBetterTransformer),
```

Reply:
Yeah. The implementation are similar. Replaced it and it works fine.